addons/survey/static/src/scss/survey_survey_views.scss

```diff
@@ -1,5 +1,5 @@
 // KANBAN VIEW
-.o_survey_survey_view_kanban {
+.o_survey_survey_view_kanban .o_kanban_renderer {
```

Comment:
maybe remove class (survey_survey lol) and use js_class instead (survey_view_kanban)

--> in other simple cases: do that too?